pkg/docker/config/config.go

```diff
@@ -51,11 +51,12 @@ var (
 	ErrNotSupported = errors.New("not supported")
 )
 
-// SetAuthentication stores the username and password in the credential helper or file
-func SetAuthentication(sys *types.SystemContext, registry, username, password string) error {
+// SetCredentials stores the username and password in the credential helper or file
```

Comment:
Please document what is the return value, consistently with what the function ends up returning (and don’t promise it to be a path).

Reply:
@mtrmac resolved in latest commit.